core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -160,7 +160,7 @@ protected[controller] class RestAPIVersion(apipath: String, apiversion: String)(
             sendCorsHeaders {
                 (pathEndOrSingleSlash & get) {
                     complete(OK, info)
-                } ~ authenticate(basicauth) {
+                } ~ (authenticate(basicauth) | authenticate(certificateAuth)) {
```

Comment:
curl: (6) Could not resolve host: GET; Name or service not known
did you curl work in the second case?

i’m not suggesting bypassing the controller
send the request to nginx
but configure nginx to not check the certificate
i claim the controller will respond as if you provided an auth key for the subject named in the header